Introduction to Kotlin Multiplatform/05_reuse_code.md

```diff
@@ -20,7 +20,7 @@ fun main() {
 
 ## Using Kotlin/Common
 
-The [kotlin-multiplatform](/docs/reference/building-mpp-with-gradle.html) plugin
+The [kotlin-multiplatform](https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html) plugin
```

Comment:
The [kotlin-multiplatform](https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html) plugin
supports the Kotlin/Common code and is a way to share code
between different targets. Let's use it now to avoid duplication
of the JVM server address.